manifests/init.pp

```diff
@@ -26,70 +26,39 @@
 #   }
 #
 # @param manage_service Disable/Enable the management of the ferm daemon
-#   Default value: false
```

Comment:
It would be nice if we could keep the allowed value?

Reply:
The data type defined the allowed values.
And puppet-strings generates doc by giving the data type of a parameter and so the allow values.

I do not understand the interest to manually maintain what is generated by puppet-strings. By this way we have to maintain two informations that can be not synchronized. 

One exception is when the default value is put into hiera data of the module. In this case puppet-strings does not  include the values contained in hiera data.